src/site/content/en/authors/index.njk

```diff
@@ -27,7 +28,7 @@ pagination:
   <section class="w-grid">
     <div class="w-grid__columns w-grid__columns--three" role="list">
       {% for element in paged.elements %}
-        {% AuthorCard element %}
+        {{ authorcard(element) }}
       {% endfor %}
```

Comment:
I'm curious if this would read better as:

```njk
{% for author in paged.elements %}
  {{ authorcard(author) }}
{% endfor %}
```

Since we're inside of authors/index.njk, it seems like there's enough context to use the word 'author' in the argument.